.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,61 +1,40 @@
 ---
 name: ✨ Feature request
 about: Suggest an idea for this project
-
 ---
-
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
+<!--- Verify first that your feature was not already discussed on GitHub -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
 ##### SUMMARY
-<!--- Explain the problem briefly -->
+<!--- Describe the new feature/improvement briefly below -->
 
 ##### ISSUE TYPE
- - Feature Idea
+- Feature Idea
 
 ##### COMPONENT NAME
-<!--- Insert, BELOW THIS COMMENT, the name of the module, plugin, task or feature.
-Do not include extra details here, e.g. "vyos_command" not "the network module vyos_command" or the full path-->
+<!--- Write the short name of the new feature, module, plugin or task below -->
 
 ##### ANSIBLE VERSION
-<!--- Paste, BELOW THIS COMMENT, verbatim output from "ansible --version" between quotes below -->
-```
+<!--- Paste verbatim output from "ansible --version" between quotes -->
+```paste below
 
 ```
 
 ##### CONFIGURATION
-<!--- If using Ansible 2.4 or above, paste, BELOW THIS COMMENT, the results of "ansible-config dump --only-changed"
-Otherwise, mention any settings you have changed/added/removed in ansible.cfg
-(or using the ANSIBLE_* environment variables).-->
+<!--- Paste verbatim output from "ansible-config dump --only-changed" between quotes -->
+```paste below
+
+```
 
 ##### OS / ENVIRONMENT
-<!--- Mention, BELOW THIS COMMENT, the OS you are running Ansible from, and the OS you are
-managing, or say "N/A" for anything that is not platform-specific.
-Also mention the specific version of what you are trying to control,
-e.g. if this is a network bug the version of firmware on the network device.-->
+<!--- Provide all relevant information below, eg. target OS versions, network device firmware, etc. -->
```

Comment:
s/eg./e.g./

Reply:
Mabe have smth like
```
Controller host OS name and version: ...
Controller host Python version: ...

Target host OS name and version: ...
Target host Python version: ...

Target network device vendor, model, firmware version, config: ...
```

Reply:
If the **ANSIBLE VERSION** and **CONFIGURATION** are not relevant, this isn't either.